vitrina/smart_contracts/models.py

```diff
@@ -114,7 +116,7 @@ def generate_odrl_jsonld(self):
         scopes = list(self.scopes.values_list("scope", flat=True))
         return {
             "@context": {"@vocab": "http://www.w3.org/ns/odrl.jsonld", "ex": "http://example.org/vocab#"},
-            "uid": f"uuid:{self.pk}",
+            "uid": f"https://data.gov.lt/id/datasets/gov/vssa/isris/dcat/Agreement/{self.pk}",
```

Comment:
What's this id part? It doesn't conform to UAPI urls
```suggestion
            "uid": f"https://data.gov.lt/ID/datasets/gov/vssa/isris/dcat/Agreement/{self.pk}",
```